crates/core/src/client/mod.rs

```diff
@@ -112,23 +108,16 @@ impl<P: Provider + Send + Sync> KakarotEthApi<P> for KakarotClient<P> {
 
     /// Returns the result of executing a call on a ethereum address for a given calldata and block
     /// without creating a transaction.
-    async fn call_view(
-        &self,
-        ethereum_address: Address,
-        calldata: Bytes,
-        block_id: BlockId,
-    ) -> Result<Bytes, EthApiError<P::Error>> {
+    async fn call_view(&self, to: Address, calldata: Bytes, block_id: BlockId) -> Result<Bytes, EthApiError<P::Error>> {
```

Comment:
I'm wondering if we should rename this. `call` or `eth_call` maybe? `call` already means `view` in the starknet and evm context

Reply:
I would name it `call` to follow the eth rpc convention